launch/tier4_perception_launch/launch/object_recognition/detection/detector/lidar_dnn_detector.launch.xml

```diff
@@ -6,8 +6,6 @@
 
   <!-- Lidar detector centerpoint parameters -->
   <arg name="centerpoint_model_name" default="centerpoint_tiny"/>
-  <arg name="centerpoint_model_path" default="$(var data_path)/lidar_centerpoint"/>
-  <arg name="lidar_model_param_path" default="$(find-pkg-share lidar_centerpoint)/config"/>
```

Comment:
This seems to be still used in the launcher

Reply:
@knzo25 Thank you for your review. 
I think `centerpoint_model_path` and `lidar_model_param_path` are already defined in the upper layer launcher [here](https://github.com/autowarefoundation/autoware.universe/blob/9f915019c9eaa4a1ecd0b20dc77c6c603ceac407/launch/tier4_perception_launch/launch/perception.launch.xml#L40)  and [here](https://github.com/autowarefoundation/autoware.universe/blob/9f915019c9eaa4a1ecd0b20dc77c6c603ceac407/launch/tier4_perception_launch/launch/perception.launch.xml#L186) so that we can remove the definition with default value here.

Reply:
Anyway, I fixed here https://github.com/autowarefoundation/autoware.universe/pull/6591/commits/c959c305562632e6dcea14012d4f42291e64ee75#diff-174cd66ea65cd31445e77661abe9e17d4882333ed8133ee5d73a08029be0e8a2R8-R10